fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/PartitionAvailability.java

```diff
@@ -21,35 +21,38 @@
 import javax.validation.constraints.NotNull;
 
 /**
- * An implementation of Availability that unions same columns from Druid tables together, in a way that the same column
- * in 2 different availabilities contains different values. Therefore if only values in one of the availabilities is
- * requested we only need to consider that table's availability.
+ * An implementation of Availability which merges distinct record rows from multiple availabilities, using the
+ * intersection of the availability of the child availabilities, taking into consideration only children whose rows
+ * are expected to participate.
+ * The default assumption is that the schemas of all tables will be the same, and if they are not, outer join
```

Comment:
If this is supposed to be a paragraph break, it should have a `<p>` on a line all by itself between the paragraphs. Otherwise it renders as 1 big paragraph.